src/python-frontend/python_converter.cpp

```diff
@@ -1,4 +1,5 @@
 #include "python-frontend/python_converter.h"
+#include "python_frontend_types.h"
 #include "util/std_code.h"
 #include "util/c_types.h"
 #include "util/arith_tools.h"
```

Comment:
Sorry, I'm seeing this just now: Could you instead of including via the quoted form `"..."` please use the angle brackets form `<...>`? The `src` directory is part of the include paths during building, there is no need to use the quoted form. (applies to all the includes for the python frontend).

Reply:
Done.
It was considering this: https://gcc.gnu.org/onlinedocs/gcc-2.95.3/cpp_1.html#SEC6
"" for own headers and <> for system headers.

Reply:
> "" for own headers and <> for system headers.

That's a useful interpretation when the project is organized such that the "own" headers are internal and placed alongside the sources in the same directory (or a sub-directory thereof). This is nearly, but not fully, the case for ESBMC. In fact the "..." variant just adds the current directory as a first search path (and also considers the ones given by `-iquote`, but we don't use that compiler option). But for ESBMC's convention that wouldn't help: `#include "util/std_code.h"` refers to the src/util sub-directory, not a subdir of the current one. The convention is to specify the full path under src/ and use the angle-bracket form.

See items 1.-6. at https://gcc.gnu.org/onlinedocs/gcc/Directory-Options.html for how the search paths are built and used.